deployment/terraform/assets/datasource.yaml

```diff
@@ -14,6 +14,7 @@ datasources:
   editable: true
   jsonData:
     timeInterval: "5s"
+    httpMethod: "GET"
```

Comment:
:eyes:

Reply:
Unsure about this one, but it wasn't working for me until I set the method to get (either manually in the web UI or by updating this file)